app/assets/stylesheets/components/_card_list.css.sass

```diff
@@ -3,7 +3,7 @@
 
   .Discussion,
```

Comment:
*Moving the margin to .Discussion and .CommentCard*
This is not a solution, because the elements would be positioning themselves, which is not a property of a reusable component.

*.CardList:nth-child(n)*
This one is in the right direction although you'd be coupled to the markup order.

So the best solution would be creating some descendant members that would wrap `.Discussion` and `.CommentCard` and position them